articles/sentinel/azure-sentinel-billing.md

```diff
@@ -198,6 +199,12 @@ With ADX, you can store data at a lower price, but still explore the data using
 
 For more information, see [Integrate Azure Data Explorer for long-term log retention](store-logs-in-azure-data-explorer.md).
 
+#### Use Data Collection Rules for your Windows Security Events
+
+The [Windows Security Events connector](https://docs.microsoft.com/en-us/azure/sentinel/connect-windows-security-events?tabs=LAA) lets you stream security events from any Windows server (physical or virtual, on-premises or in any cloud) connected to your Azure Sentinel workspace. The new version of this connector is currently in preview and is based on the new Azure Monitor Agent (AMA). The Azure Monitor agent uses Data collection rules (DCR) to define the data to collect from each agent. Data collection rules let you manage collection settings at scale while still allowing unique, scoped configurations for subsets of machines. See [Configure data collection for the Azure Monitor agent](https://docs.microsoft.com/en-us/azure/azure-monitor/agents/data-collection-rule-azure-monitor-agent) for more details.
```

Comment:
@nayef-yassin please update with these edits:

The [Windows Security Events connector](/azure/sentinel/connect-windows-security-events?tabs=LAA) enables you to stream security events from any Windows server that's connected to your Azure Sentinel workspace, including physical, virtual, or on-premises servers, or in any cloud. This connector includes support for the Azure Monitor Agent (AMA), which uses data collection rules (DCR) to define the data to collect from each agent. 

Data collection rules enable you to manage collection settings at scale, while still allowing unique, scoped configurations for subsets of machines. For more information, see [Configure data collection for the Azure Monitor agent](/azure/azure-monitor/agents/data-collection-rule-azure-monitor-agent).

Besides for the predefined sets of events that you can select to ingest, such as *All events*, *Minimal*, or *Common*, data collection rules enable you to build custom filters and select specific events to ingest. The Azure Monitor Agent uses these rules to filter the data at the source, and then ingest only the events you've selected, while leaving everything else behind. Selecting specific events to ingest can help you optimize your costs and save more.